css/main.css

```diff
@@ -1,3 +1,667 @@
+  width: 17px;
+  height: 17px;
+}
+.resume ul li {
+  display: flex;
+  align-items: center;
+}
+
+/* Main - Project Page */
+.projects {
+  background-color: var(--beige-dark);
+  padding: 4rem 0rem 2rem 2rem;
+  line-height: 1.7;
+}
+.projects .title {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.projects .title ~ p {
+  margin-top: 1.5rem;
+  margin-right: 0.5rem;
+}
+
+.projects .pro-articles {
+  width: 97%;
+  margin-top: 3rem;
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+}
+.projects .pro-articles .article {
+  background-color: var(--white);
+  /* padding-top: 1.3rem; */
+  display: flex;
+  flex-direction: column;
+}
+.projects .pro-articles .article .col {
+  flex: 1;
+}
+.projects .pro-articles .article .col1 {
+  padding-top: 1.3rem;
+}
+
+.projects .article .article-title {
+  border-left: 12px solid var(--blue-royal);
+  padding-left: 1.2rem;
+}
+.projects .article .article-title ~ p {
+  padding: 2rem 1.2rem 1.5rem 1.2rem;
+}
+.projects .article img {
+  width: 100%;
+}
+
+/* Main - Contact Page */
+.contact {
+  background-color: var(--beige-dark);
+  padding: 2.5rem 0.5rem 2rem 2rem;
+  line-height: 2;
+}
+.contact .title {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.contact form {
+  background-color: var(--white);
+  padding: 2rem 0.75rem;
+  margin-top: 2.5rem;
+}
+.contact form div {
+  display: flex;
+  flex-direction: column;
+  margin-top: 0.5rem;
+}
+.contact form input {
+  height: 30px;
+  padding-left: 1rem;
+  border: none;
+  border-bottom: 2px solid var(--black);
+}
+.contact form input:focus {
+  outline: none;
+  border-bottom: 2px solid var(--blue-royal);
+}
+.contact form .form-btn {
+  margin-top: 1.5rem;
+}
+.contact form a {
+  /* display: inline-block; */
+  margin: 0 auto;
+  padding: 0.2rem 3rem;
+}
+
+/* Footer section */
+footer {
+  padding: 2rem 2rem;
+}
+footer .row1 {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: auto;
+  grid-gap: 1rem;
+  margin-bottom: 1rem;
+}
+footer .first-three-div .col3 .social-icons {
+  display: flex;
+  flex-direction: row;
+  margin-top: 0.25rem;
+}
+footer .first-three-div .col3 .social-icons li {
+  margin-right: 1rem;
+}
+footer .row2 {
+  display: flex;
+  justify-content: space-between;
+}
+
+.circle {
+  display: inline-block;
+  vertical-align: middle;
+}
+.circle a {
+  display: inline-block;
+  border-radius: 50%;
+  background-color: var(--blue-royal);
+}
+.top {
+  display: inline-block;
+  width: 1.5em;
+  height: 2em;
+  border-radius: 50%;
+  margin-right: 0.75em;
+}
+
+.top:after {
+  content: "";
+  display: inline-block;
+  margin-top: 0.8em;
+  margin-left: 0.85em;
+  width: 0.5em;
+  height: 0.5em;
+  border-top: 0.1em solid var(--white);
+  border-right: 0.1em solid var(--white);
+  -moz-transform: rotate(-45deg);
+  -webkit-transform: rotate(-45deg);
+  transform: rotate(-45deg);
+}
+
+.hide-me {
+  animation: cssAnimation 0s 5s forwards;
```

Comment:
not sure what was the initial intention but you might wanna delete the animation. otherwise arrow button get disappear after 5 seconds 🥹